_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/scripts/js/rhd/5-sso.js

```diff
@@ -32,6 +32,20 @@ app.sso = function () {
                 $('section.contributors-banner, .shown-after-login, li.logged-in, [data-audience="authenticated"]').show();
                 $('li.login a, a.keycloak-url').attr("href", keycloak.createAccountUrl());
 
+                // Remove unauthenticated links from an on_page_navigation assembly, if one exists on the page.
```

Comment:
Since this is written twice, once again on line 93,,  can you change this to a function and pass the state to it, such as 

`toggleOnPageNav(Keycloak.authenticated)`

Which will reduce the # of lines here by 16.

I have a pending PR on UnpublishedState that does this with `showAudienceSelectionContent`,  which may need to be consolidated depending on which one of these PRs goes up first.